Server/src/assets/logo.ts

```diff
@@ -0,0 +1,2 @@
+export const logo: string =
```

Comment:
Easier way to manage importing an image without needing `fs` by just having it as a string which can be useful when setting default images in the codebase